src/Panel/SearchResultsPanel/SearchResultsPanel.less

```diff
@@ -0,0 +1,65 @@
+.search-wrapper {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+
+    .ant-input {
+      background-color: white;
+    }
+
+  }
+
+  .search-result-div {
+    width: 50vw;
+    position: absolute;
+    top: 47px;
+    z-index: 9999;
+
+    .ant-collapse {
+      max-height: 400px;
+      overflow-y: auto;
+
+      .ant-list div.result-text {
+        padding: 0 10px;
+      }
+
+      .ant-collapse-header {
+        padding: 5px 0 5px 40px;
+        background-color: #E9E9E9;
```

Comment:
Can we remove the hardcoded color values from here? The antd defaults, which can be overridden by the certain project if needed, will be applied instead

Reply:
Removed in 92f8c02e5